.gitignore

```diff
@@ -62,3 +62,4 @@ typings/
 
 # Build artifacts
 build/
+dist/
```

Comment:
Is streamable published to npm as a bundle? If so this is fine. If it isn't though then I can't manage it as a dependency

Reply:
Yep, it's published: https://www.npmjs.com/package/oyster-streamable

Reply:
It's a bit out of date. Can we add a hook to serve the bundle and the babel build to npm when a there is a new commit to master?

Reply:
Do we want to auto publish like that?

I don't have much experience with npm publishing, but is there a way to publish to a dev or beta branch instead so that we aren't incrementing releases so often?

Reply:
I think [semantic release](https://www.npmjs.com/package/semantic-release) is the most popular auto deployment option. I've never used auto deployment before but it seems like it would be worthwhile